examples/specs/rect_mosaic_simple_future.vl.json

```diff
@@ -0,0 +1,106 @@
+{
+ "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+ "data": {
+   "url": "data/cars.json"
+ },
+ "transform": [
+   {
+     "aggregate": [
+       {
+         "op": "count",
+         "as": "count_*"
+       }
+     ],
+     "groupby": [
+       "Origin",
+       "Cylinders"
+     ]
+   },
+   {
+     "stack": "count_*",
+     "groupby": [],
+     "as": [
+       "stack_count_Origin1",
+       "stack_count_Origin2"
+     ],
+     "offset": "normalize",
+     "sort": [{
+       "field": "Origin",
+       "order": "ascending"
+     }]
+   },
+   {
+     "window": [
+       {
+         "op": "min",
+         "field": "stack_count_Origin1",
+         "as": "x"
+       },
+       {
+         "op": "max",
+         "field": "stack_count_Origin2",
+         "as": "x2"
+       }
+     ],
+     "groupby": [
+       "Origin"
+     ],
+     "frame": [
+       null,
+       null
+     ]
+   },
+   {
+     "stack": "count_*",
+     "groupby": [
+       "Origin"
+     ],
+     "as": [
+       "y",
+       "y2"
+     ],
+     "offset": "normalize",
+     "sort": [{
+       "field": "Cylinders",
+       "order": "ascending"
+     }]
+   }
+ ],
+ "mark": {"type": "rect"},
+ "encoding": {
+   "y": {
```

Comment:
Can you help transpose these so it's consistent with other ones?  (map y: y, y2:y2)